EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp

```diff
@@ -19,7 +19,7 @@ CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
 
-
+/*
```

Comment:
Is there a reason these are all commented out? If so, please create a todo tied to a github issue to deal with that reason. If not, please restore them.

Reply:
they rely on a mock I deleted when cleaning regions. mappers are next on my list so I'll be revising this in the next PR.